### ISSUE TITLE
pip 21.0.1 error  "amuse-framework>=12.1.3[MPI]"

### ISSUE THREAD

**Issue by aatrani** [amusecode/amuse#774]
- Error when installing amuse using pip 21.0.1  
> pip install amuse --user

The full error message is the following:
```  Using cached amuse-12.1.3.tar.gz (11 kB)
  Installing build dependencies ... error
  ERROR: Command errored out with exit status 1:
   command: /usr/bin/python3 /usr/lib/python3.9/site-packages/pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-_el7vg8v/overlay --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- 'amuse-framework>=12.1.3[MPI]' 'amuse-athena>=12.1.3' 'amuse-bhtree>=12.1.3' 'amuse-brutus>=12.1.3' 'amuse-bse>=12.1.3' 'amuse-capreole>=12.1.3' 'amuse-evtwin>=12.1.3' 'amuse-fastkick>=12.1.3' 'amuse-fi>=12.1.3' 'amuse-fractalcluster>=12.1.3' 'amuse-gadget2>=12.1.3' 'amuse-galactics>=12.1.3' 'amuse-galaxia>=12.1.3' 'amuse-halogen>=12.1.3' 'amuse-hermite>=12.1.3' 'amuse-hop>=12.1.3' 'amuse-huayno>=12.1.3' 'amuse-kepler>=12.1.3' 'amuse-kepler-orbiters>=12.1.3' 'amuse-mameclot>=12.1.3' 'amuse-mercury>=12.1.3' 'amuse-mmams>=12.1.3' 'amuse-ph4>=12.1.3' 'amuse-seba>=12.1.3' 'amuse-secularmultiple>=12.1.3' 'amuse-smalln>=12.1.3' 'amuse-sphray>=12.1.3' 'amuse-sse>=12.1.3' 'amuse-twobody>=12.1.3'
       cwd: None
  Complete output (1 lines):
  ERROR: Extras after version '>=12.1.3[MPI]'.
  ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/ea/73/1c9294b46df3f05b0e47fb10be338f6a60fdb49aa82aa36ec25df902886d/amuse-12.1.3.tar.gz#sha256=803016dde50df4353b1fd181430b0420e1c07c1ef0fd596aad02da39c678b67a (from https://pypi.org/simple/amuse/). Command errored out with exit status 1: /usr/bin/python3 /usr/lib/python3.9/site-packages/pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-_el7vg8v/overlay --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- 'amuse-framework>=12.1.3[MPI]' 'amuse-athena>=12.1.3' 'amuse-bhtree>=12.1.3' 'amuse-brutus>=12.1.3' 'amuse-bse>=12.1.3' 'amuse-capreole>=12.1.3' 'amuse-evtwin>=12.1.3' 'amuse-fastkick>=12.1.3' 'amuse-fi>=12.1.3' 'amuse-fractalcluster>=12.1.3' 'amuse-gadget2>=12.1.3' 'amuse-galactics>=12.1.3' 'amuse-galaxia>=12.1.3' 'amuse-halogen>=12.1.3' 'amuse-hermite>=12.1.3' 'amuse-hop>=12.1.3' 'amuse-huayno>=12.1.3' 'amuse-kepler>=12.1.3' 'amuse-kepler-orbiters>=12.1.3' 'amuse-mameclot>=12.1.3' 'amuse-mercury>=12.1.3' 'amuse-mmams>=12.1.3' 'amuse-ph4>=12.1.3' 'amuse-seba>=12.1.3' 'amuse-secularmultiple>=12.1.3' 'amuse-smalln>=12.1.3' 'amuse-sphray>=12.1.3' 'amuse-sse>=12.1.3' 'amuse-twobody>=12.1.3' Check the logs for full command output.
````

The main error seems to arise from the syntax:
> amuse-framework>=12.1.3[MPI]  

that was deprecated with the recent updates of pip

Then pip tries to install older amuse versions, eventually settling on version amuse-12.1.0, which is the last version not to have the [MPI] tag.

**Comment by ipelupessy** [comment]
@rieder was this fixed?

**Comment by rieder** [comment]
In source yes but we haven't released a new package yet that includes the fix...

**Comment by rieder** [comment]
Fixed in 2021.7.0
